subprojects/test-runner/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/InstrumentationTestsTask.kt

```diff
@@ -205,6 +206,9 @@ public abstract class InstrumentationTestsTask @Inject constructor(
             saveTestArtifactsToOutputs = experiments.saveTestArtifactsToOutputs,
             useLegacyExtensionsV1Beta = experiments.useLegacyExtensionsV1Beta,
             sendPodsMetrics = experiments.sendPodsMetrics,
+            needForward = project.getOptionalStringProperty(
```

Comment:
You should add this to gradle Extention. And choose better name. `adbDevicePortForward` for example